packages/lib/timeFormat.ts

```diff
@@ -18,11 +18,19 @@ export const setIs24hClockInLocalStorage = (is24h: boolean) =>
 
 export const getIs24hClockFromLocalStorage = () => {
   const is24hFromLocalstorage = localStorage.getItem(is24hLocalstorageKey);
-  if (is24hFromLocalstorage === null) return null;
```

Comment:
Single line returns are preferred for simple comparisons 🙏

Reply:
Also we try to avoid git diffs for stylistic choices

Reply:
Oh sorry! It's a habit to remove them as I've found they are often responsible for bugs that go unnoticed since it's harder to parse/spot. I can revert that shortly.